src/components/Profile/ProfileContainer.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import axios from "axios";
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+
+import { setUserProfile } from "../../redux/actionCreators";
+
+import Profile from "./Profile";
+
+class ProfileContainer extends React.Component {
+  componentDidMount() {
+    const userId = this.props.match.params.userId;
```

Comment:
If statement check of "userId" is basically needed to set the default value of "userId".
Consider instead of using if statement doing the defaulting by 
`const userId = this.props.match.params.userId || 9;` which does the same.
Operator "||" is logical OR.

Reply:
Отлично. Сделано